website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -24,22 +24,22 @@ By default, if you define an icon, it is placed in the leading position (before
 <Hds::Button @text="Copy to clipboard" @icon="clipboard-copy" @iconPosition="trailing" />
 ```
 
-### Icon-only button
+### Icon-only Button
 
 !!! Info
 
-To add a tooltip to an icon-only button, here’s an example of how to do it in an accessible way: [Accessible Button Tooltip Pattern](https://codepen.io/melsumner/pen/bGGdmMV).
+To add a tooltip to an icon-only Button, here’s an example of how to do it in an accessible way: [Accessible Button Tooltip Pattern](https://codepen.io/melsumner/pen/bGGdmMV).
 !!!
 
-If you would like to create an icon-only button, set `@isIconOnly` to `true`. Note that you still have to define the `@text` value; it will be used as the `aria-label` attribute value on the `button` element.
+If you would like to create an icon-only Button, set `@isIconOnly` to `true`. Note that you still have to define the `@text` value; it will be used as the `aria-label` attribute value on the `Button` element.
 
 ```handlebars
 <Hds::Button @text="Copy to clipboard" @icon="clipboard-copy" @isIconOnly={{true}} />
 ```
 
 ### Color
 
-There are four available colors for a button: `primary`, `secondary`, `tertiary`, and `critical`. The default is `primary`. To use a different color, declare another value for `@color`:
+There are four available colors for the Button: `primary`, `secondary`, `tertiary`, and `critical`. The default is `primary`. To use a different color, declare another value for `@color`:
```

Comment:
```suggestion
There are four available colors for Buttons: `primary`, `secondary`, `tertiary`, and `critical`. The default is `primary`. To use a different color, declare another value for `@color`:
```
not sure about the definitive article usage here, maybe we can match what we have on the size section?

Reply:
I think it would be okay to say "the Button" here because it's referring to the component. If we were saying "four available colors for buttons" then it would be generic and lowercased.

Reply:
Thanks, both. I think I will go with "the Button" wording for both. (I had considered making that change before but wasn't certain.)